cms/elm/Ports.elm

```diff
@@ -19,3 +19,6 @@ port swipe : (String -> msg) -> Sub msg
 
 
 port tipSwipe : (String -> msg) -> Sub msg
+
+
+port clickScroll : () -> Sub msg
```

Comment:
This is one part I know must be wrong. I know I should be listening for *something* from JS, but I am not sure what. What I am doing JS side is when clicking on the search button and calling the function to smooth scroll to results. However `window.scrollTo` doesn't have a return value so I'm not sure what to send back, or how I can rewrite the JS so that something can be sent back.